src/api/app/views/webui2/shared/_repositories_flag_table_column.html.haml

```diff
@@ -1,9 +1,19 @@
 :ruby
+  is_flag_set_by_user = !flag.new_record?
   title = flag.effective_status == 'disable' ? 'Disabled' : 'Enabled'
-  title += ' (default)' if flag.new_record?
-  content = String.new(render(partial: 'shared/flag_popover', locals: { flag: flag, project: project, package: package, remote: !flag.has_children }))
+  title += is_flag_set_by_user ? ' (set by user)' : ' (calculated)'
```

Comment:
To be consistent with this sentence from the view
> You can configure individual flags for this package here. The repositories are inherited from the project home:Admin

We should have `' (inherited)'` or even better, `' (inherited from project)'`.

Reply:
Is 'calculated' not always inherited. If the user sets the All/All flag for instance then the single flags are calculated based on that flag... Is kind of difficult to understand maybe but it is how it works right now.

Summarizing.... a flag value can come from itself when the users sets it, or can come from an All column or row, or form an All/All flag... or from the project in the same combination (I mean with the All columns/rows/table flag value) orrr from the default of the app.